docs/integrations.md

```diff
@@ -6,7 +6,7 @@ Semgrep integrates into the development flow end-to-end, from code conception in
 
 # Continuous integration (CI)
 
-The following instructions use [Semgrep CI](https://github.com/returntocorp/semgrep-action) and require a free [Semgrep Community](https://semgrep.dev/manage) or paid Semgrep Team account. `SEMGREP_DEPLOYMENT_ID` and `SEMGREP_APP_TOKEN` information is available at [Manage > Settings](https://semgrep.dev/manage/settings) after login.
+The following instructions use [Semgrep CI](https://github.com/returntocorp/semgrep-action) and require a [Semgrep App](https://semgrep.dev/manage) account (free or paid tier). `SEMGREP_DEPLOYMENT_ID` and `SEMGREP_APP_TOKEN` information is available at [Manage > Settings](https://semgrep.dev/manage/settings) after login.
```

Comment:
Throughout the docs we use the language "free or paid tier" frequently. Do you feel the reminder is necessary?

Reply:
Probably not necessary to use it quite so frequently.

Reply:
@dlukeomalley ugh how do I accept your suggestion here?